ansible/roles/config/templates/config.toml.tmpl

```diff
@@ -92,14 +91,17 @@ imports = ["/etc/containerd/conf.d/*.toml"]
         privileged_without_host_devices = false
       [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
         [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
-          runtime_type = "io.containerd.runc.v1"
+          runtime_type = "io.containerd.runc.v2"
           runtime_engine = ""
           runtime_root = ""
           privileged_without_host_devices = false
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+            systemd_cgroup = true
```

Comment:
```suggestion
            SystemdCgroup = true
```

Reply:
I think we want this, based on some testing

Reply:
Confirmed independently. I ran `kubeadm init`, then checked the configuration of the various control-plane containers that run in Static Pods:

Using `systemd_cgroup`:
```shell
# crictl ps -q | xargs -n1 crictl inspect | grep systemd_cgroup
```

Using `SystemdCgroup`
```shell
# crictl ps -q | xargs -n1 crictl inspect | grep systemd_cgroup
      "systemd_cgroup": true
      "systemd_cgroup": true
      "systemd_cgroup": true
      "systemd_cgroup": true
      "systemd_cgroup": true
```